Source/WebInspectorUI/UserInterface/Views/ConsoleMessageView.js

```diff
@@ -159,6 +162,37 @@ WI.ConsoleMessageView = class ConsoleMessageView extends WI.Object
         this._repeatCountElement.textContent = Number.abbreviate(count);
     }
 
+    _renderTimestamp()
+    {
+        if (!this._timestamp) {
+            if (this._timestampElement) {
+                this._timestampElement.remove();
+                this._timestampElement = null;
+            }
+            return;
+        }
+
+        if (!this._timestampElement) {
+            this._timestampElement = document.createElement("span");
+            this._timestampElement.classList.add("timestamp");
+            this._element.insertBefore(this._timestampElement, this._element.firstChild);
+        }
+
+        let date = new Date(this._timestamp * 1000);
+        var hours = date.getHours(),
```

Comment:
Nit: `let`

Reply:
also please do a `let` for each instead of using `,`